### PR TITLE
Update PR template

### PR DESCRIPTION
**Pull request by kangkaisen**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by silverbullet233 — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,13 +10,23 @@ Fixes #issue
...
 - [ ] Tool
 
+## Is there a behavior change:
+
+- [x] This PR will involve a behavior change
+  - [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
+  - [ ] Parameter changes: default values, similar parameters but with different default values
+  - [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
+  - [ ] Feature removed
+  - [ ] Miscellaneous: updgrade & downgrade compatibility, ...
```
Comment (imay) [inline .github/PULL_REQUEST_TEMPLATE.md:20]:
```suggestion
Does this PR entail a change in behavior?

- [x] Yes, this PR will result in a change in behavior.
- [ ] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.
```

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by silverbullet233 — APPROVED** [review]

**Review by satanson — APPROVED** [review]

**Pull request merged**